From 63c4d0becc9e1aedcae4ffd0956fd1999b58b15a Mon Sep 17 00:00:00 2001
From: Max Ihlenfeldt <max@igalia.com>
Date: Tue, 7 Nov 2023 15:09:30 +0000
Subject: [PATCH] Backport "IWYU for
 chrome/browser/ui/views/dark_mode_manager_linux.h"

This fixes some build errors due to missing imports. The patch will be
included in upstream's 121 release.

Upstream-Status: Backport [https://crrev.com/c/5013111]
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 chrome/browser/ui/views/dark_mode_manager_linux.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/ui/views/dark_mode_manager_linux.h b/chrome/browser/ui/views/dark_mode_manager_linux.h
index f9bcd03..621b67b 100644
--- a/chrome/browser/ui/views/dark_mode_manager_linux.h
+++ b/chrome/browser/ui/views/dark_mode_manager_linux.h
@@ -6,11 +6,13 @@
 #define CHROME_BROWSER_UI_VIEWS_DARK_MODE_MANAGER_LINUX_H_

 #include <string>
+#include <vector>

 #include "base/gtest_prod_util.h"
 #include "base/memory/scoped_refptr.h"
 #include "base/memory/weak_ptr.h"
 #include "base/scoped_observation.h"
+#include "ui/native_theme/native_theme.h"
 #include "ui/native_theme/native_theme_observer.h"

 namespace dbus {
